.php_cs

```diff
@@ -15,6 +15,7 @@ $config = PhpCsFixer\Config::create()
         'no_unused_imports' => true,
         'ordered_imports' => true,
         'psr0' => false,
+        'single_blank_line_before_namespace' => true,
```

Comment:
@BackEndTea 

What do you think about enabling this fixer in a separate PR (similar to - for example - https://github.com/opencfp/opencfp/pull/524)? 

Personally, I like to do this in two steps:

* enable the fixer and commit (see the build fail)
* run `make cs` and commit (make the build pass)

A side effect of this is that I wait with enabling a fixer until I see a PR opened that indicates that a project could benefit from enabling a specific rule (for example, I spotted here that - while consistent in context of this PR - spacing is applied before `:` in return type declarations, but both the proposed [PSR-12](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md) as well as the default configuration for the `return_type_declaration` fixer indicate that no space should be used, so it's a good case for enabling the fixer on an existing project). 🤓